node-api/handlers/debug/routes.go

```diff
@@ -37,7 +37,7 @@ func (h *Handler) RegisterRoutes(logger log.Logger) {
 		},
 		{
 			Method:  http.MethodGet,
-			Path:    "/eth/v2/debug/beacon/states/heads",
+			Path:    "/eth/v2/debug/beacon/heads",
 			Handler: h.NotImplemented,
 		},
```

Comment:
This is the correct path https://ethereum.github.io/beacon-APIs/#/Debug/getDebugChainHeadsV2

Reply:
cc @calbera @rezbera @camembera